massa-async-pool/src/changes.rs

```diff
@@ -100,10 +103,14 @@ impl Serializer<AsyncPoolChanges> for AsyncPoolChangesSerializer {
                     self.id_serializer.serialize(id, buffer)?;
                     self.message_serializer.serialize(message, buffer)?;
                 }
-                Change::Delete(id) => {
+                Change::Activate(id) => {
                     buffer.push(1);
```

Comment:
Should we use an enum here or constants?

Reply:
Done.